### PR TITLE
1.4.0 release updates

### PR DESCRIPTION
**Pull request by corycaywood**
# Description
Update changelog date and version in package.json.

## Links
Add links to github/jira issues, design documents and other relevant resources (e.g. previous discussions, platform/tool documentation etc.)

# Checklist
- [x] I wrote/updated tests for new/changed code.
- [x] I ran `yarn sdk buildSdk` and `yarn bridge buildBridge` without issues.
- [x] I ran `yarn sample prettify` and `yarn sample build` without issues.

### CODE REVIEW

**Review by khairul-alam-licon — APPROVED** [review]
👍 👍

File: js-miniapp-sdk/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## CHANGELOG
 
-### 1.x.x (In progress)
+### 1.4.0 (2020-11-20)
```
Comment (ts-leojoseph-r) [inline js-miniapp-sdk/CHANGELOG.md:3]:
```suggestion
### 1.4.0 (2020-11-02)
```

Comment (corycaywood) [reply js-miniapp-sdk/CHANGELOG.md]: Thanks, I've updated this.

**Review by minh-rakuten — APPROVED** [review]
👍 👍

**Pull request merged**